pkg/operator/ceph/file/mirror/mirror.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2021 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package mirror for mirroring
+package mirror
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/banzaicloud/k8s-objectmatcher/patch"
+	"github.com/pkg/errors"
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"github.com/rook/rook/pkg/operator/ceph/cluster/mon"
+	"github.com/rook/rook/pkg/operator/ceph/config"
+	"github.com/rook/rook/pkg/operator/ceph/controller"
+	opcontroller "github.com/rook/rook/pkg/operator/ceph/controller"
+	"github.com/rook/rook/pkg/operator/k8sutil"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	// AppName is the ceph filesystem mirror application name
+	AppName = "rook-ceph-fs-mirror"
+	// minimum amount of memory in MB to run the pod
+	cephFilesystemMirrorPodMinimumMemory uint64 = 512
```

Comment:
We really need that much memory?

Reply:
TBH, it's too early to come up with good numbers yet, so I based this on the rbd-mirror requirements we have.